adapters/andalucia.js

```diff
@@ -0,0 +1,925 @@
+'use strict';
+
+import { REQUEST_TIMEOUT } from '../lib/constants';
+import { default as baseRequest } from 'request';
+import cheerio from 'cheerio';
+import _ from 'lodash';
+import { default as moment } from 'moment-timezone';
+import async from 'async';
+const request = baseRequest.defaults({timeout: REQUEST_TIMEOUT});
+
+exports.name = 'andalucia';
+
+export function fetchData (source, callback) {
```

Comment:
Can we get some try/catch in here like previous adapters have in case there are issues running the code block?